community/nspr/build

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh -e
+
+mkdir build inst
+
+cd build
+CFLAGS="$CFLAGS -D_PR_POLL_AVAILABLE -D_PR_HAVE_OFF64_T -D_PR_INET6 -D_PR_HAVE_INET_NTOP -D_PR_HAVE_GETHOSTBYNAME2 -D_PR_HAVE_GETADDRINFO -D_PR_INET6_PROBE" \
+../nspr/configure \
+    --prefix=/usr \
+    --disable-debug \
+    --enable-optimize \
+    --enable-64bit
+
+make
+make install
+
+# $1 is never unset
```

Comment:
This is very interesting to me though.

There are sixteen instances of `rm -rf "$1/..."` in the main repository, but only *one* (`expat`) includes a check of `${1:?}` 🤔 Just an observation.

Reply:
I'd guess shellcheck only cares about `/{usr/{bin,lib,share}}` to avoid breaking the system.

Reply:
Quite possible...